graylog2-server/src/main/java/org/graylog2/rest/models/system/contenpacks/responses/ContentPackList.java

```diff
@@ -36,8 +38,13 @@
     @JsonProperty
     public abstract Set<ContentPack> contentPacks();
 
+    @JsonProperty
+    public abstract Map<ModelId, ContentPackMetaData> contentPacksMetaData();
```

Comment:
I checked the rest of the server code base and we usually use "metadata" instead of "meta data". So I think we can change the name of the field to `contentPacksMetadata` so the generated field name in JSON is `content_packs_metadata`.